server/uno.go

```diff
@@ -74,24 +74,19 @@ func createPlayer(name string) (*model.Player, error) {
 
 func createNewGame(gameName string, creatorName string) (*model.Game, *model.Player, error) {
 	database, err := db.GetDb()
-	game, err := database.CreateGame()
 	if err != nil {
 		return nil, nil, err
 	}
 
-	creator, e := createPlayer(creatorName)
-
-	if e != nil {
-		return nil, nil, e
+	creator, err := database.CreatePlayer(creatorName)
+	if err != nil {
+		return nil, nil, err
 	}
 
-	game, err = database.JoinGame(game.ID, creator.ID)
+	game, err := database.CreateGame(gameName, creator.ID)
```

Comment:
this will throw an error, due to the := operator. same on line 81. `err` was created on line 76. You either have to name these variables uniquely, or pre-instantiate all of the other variables.

Reply:
You would think, but I haven't found that to be the case. It looks like, so long as one variable has not yet been defined, the `:=` operator works just fine.

Reply:
that.... doesn't seem to make sense to me. as the `:=` operator should attempt to initialize the variable. I'll try running this locally then to confirm.  

I know that if you do something like `if game, err := database.CraeteGame() ....etc` It should allow you later on, but that's because it defines to that local scope.

Like I said... I'll go check it out now to confirm.